DDG4/plugins/Geant4SDActions.cpp

```diff
@@ -1,5 +1,5 @@
 //==========================================================================
-//  AIDA Detector description implementation 
+//  AIDA Detector description implementation
```

Comment:
Was this file really supposed to be included in this MR ? I do not see the relationship